enterprise_access/apps/content_assignments/models.py

```diff
@@ -391,6 +398,19 @@ def add_errored_redeemed_action(self, exc):
             traceback=format_traceback(exc),
         )
 
+    def clear_pii(self):
+        """
+        Removes PII field values from this assignment.
+        """
+        self.learner_email = RETIRED_EMAIL_ADDRESS
+        self.lms_user_id = None
+
+    def clear_historical_pii(self):
+        """
+        Removes PII values from this assignment's historical records.
+        """
+        self.history.update(learner_email=RETIRED_EMAIL_ADDRESS, lms_user_id=None)  # pylint: disable=no-member
+
```

Comment:
This mostly copies what we do in [license-manager](https://github.com/openedx/license-manager/blob/cfca89f003707db1d2d2a083cd4f7526808e1aae/license_manager/apps/subscriptions/models.py#L1039-L1053), except here we set to a tombstone-style email address.